File: captcha.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.captcha;

import java.util.Random;

/**
 *
 * @author Cs Koushik
 */
public class captcha 
{
    public String cap()
    {
        int i='Z';
        int l=1;
        int k=1;
        String s="";
        Random r=new Random();
        while(l<=3 || k<3)
        {
            int c=r.nextInt(i);
            if(c>'A'&& c<='Z')
            {
                s+=(char)c;
                l++;
            }
            else if(k<3)
            {
                s+=String.valueOf(r.nextInt(9));
                k++;
            }
        }
        return s;
    }
    
}

File: captcha.py
from random import randint
from random import shuffle
i='A'
s=ord(i)
j='Z'
t=ord(j)
l=1
n=[]
c=""
while l<=3:
    r=randint(s,t)
    n.append(chr(r))
    l=l+1
l=1
while l<3:
    n.append(str(randint(0,9)))
    l=l+1
shuffle(n)
for i in n:
    c=c+i
print('The Captcha is ',c)
